browser/ui/webui/brave_wallet/wallet_page_ui.cc

```diff
@@ -70,6 +70,13 @@ WalletPageUI::WalletPageUI(content::WebUI* web_ui)
       base::make_span(kBraveWalletPageGenerated, kBraveWalletPageGeneratedSize),
       IDR_WALLET_PAGE_HTML);
   source->AddString("braveWalletLedgerBridgeUrl", kUntrustedLedgerURL);
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::ImgSrc,
+      base::JoinString(
+          {"img-src", "'self'", "chrome://resources",
+           "chrome://erc-token-images", "chrome://favicon", "chrome://image",
+           "https://assets.cgproxy.brave.com", base::StrCat({"data:", ";"})},
```

Comment:
I just went through this with @samartnik on the chromium upgrade. I don't think we needed "chrome://favicon", "chrome://image", "https://assets.cgproxy.brave.com" at least on android. could you try removing those?

data: might not be needed either.

Reply:
see https://github.com/brave/brave-core/pull/22320/files#diff-0e0eb6acfeba674a0c6a698e22c04773ae563f8629ac3841fc1ec59b6d3e5b02

Reply:
updated: e1df221

Reply:
Re-added chrome://image since it is used when loading custom network logo: https://github.com/brave/brave-core/blob/ff30868185b37bd8c38421efda98ac429ef50661/components/brave_wallet_ui/components/shared/create-network-icon/index.tsx#L94